### ISSUE TITLE
bug: Auto creates a lot of anonymous functions

### ISSUE THREAD

**Issue by facu-carri** [aidenybai/million#501]
**Million create amount of anonymous functions**
Step 1: Create a block in a component
```jsx
import { block } from 'million/react'
const EmotionBlock = block(() => {
    return <div>a</div>;
 })
```
What you get is...
```bash
<_anonymous$299> was automatically optimized. We estimate reconciliation to be ~100% faster.
Found 1 nodes, 0 attributes, 0 texts, 0 components, and 0 expressions. (1 - 0) / (1 + 0) = ~100%
```
![imagen](https://github.com/aidenybai/million/assets/43284046/37c6573d-fba9-4baf-81d4-f9670b69b955)
### Edit:
I remove {auto:true} from next.config.js and gets solved
But dont throw any optimized msg

**Comment by aidenybai** [comment]
i think this is calculating it right from a reconciliation standpoint. react will need to diff the `div` and `a` elements and million doesn't need to do any diff here at all. hence 100% improvement

**Comment by aidenybai** [comment]
Fixed in v2.5.4-beta.2. if you are using RSC, make sure to enable rsc as so: `auto: { rsc: true }` @facu-carri

**Comment by tobySolutions** [comment]
Thank you for this @facu-carri. You can check out the latest version for your fixes. The team will make documentation available for automatic mode very soon.

https://www.youtube.com/watch?v=Bs_Ouo-3OvE
